main.py

```diff
@@ -29,6 +59,13 @@ def convert():
     json = request.get_json(force=True)
     return perform_notebook_conversion(json)
 
+@app.route('/api/convert/rmd', methods={'POST'})
```

Comment:
So now we have `/api/convert` for ipynb and `/api/convert/rmd` for Rmd.

Should I change the former to `/api/convert/ipynb`? 🤔